syntax = "proto3";
package appmesh;
option go_package = "./;main";

message Request {
  string uuid = 1;
  string http_method = 2;
  string request_uri = 3;
  string client_address = 4;
  bytes http_body = 5;
  map<string, string> headers = 6;
  map<string, string> querys = 7;
}
